src/test/regress/sql/multi_view.sql

```diff
@@ -0,0 +1,146 @@
+--
+-- MULTI_VIEW
+--
+
+-- This file contains test cases for view support. It verifies various
+-- Citus features: simple selects, aggregates, joins, outer joins
+-- router queries, single row inserts, multi row inserts via insert
+-- into select, multi row insert via copy commands.
+
+-- These are the settings that lineitem_hash_part and orders_hash_part tables
+-- are created with.
+SET citus.shard_count to 4;
+SET citus.shard_replication_factor to 1;
```

Comment:
later in the test I create an empty distributed table and test insert into select. Tables must be colocated in order for that to work.

create_distributed_table did not have colocated_with support back then.
I will remove this and use colocate with.